rootfs/usr/local/bin/build-kube-cfg

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Seems like this is `kops` specific. Let's rename to `build-kops-kubecfg`

Reply:
Also, rather than `kube_config.yaml` lets call it `kubecfg.yaml`, since `kops` seems to consistently call it `kubecfg`.  (unless you can point me to the inspiration for calling it `kube_config.yaml`)

Reply:
https://github.com/kubernetes/kops/blob/master/docs/cli/kops_export_kubecfg.md